cypress/integration/navigations.ts

```diff
@@ -0,0 +1,58 @@
+import { IFRAME_SELECTOR } from '../support/constants';
+
+describe('navigations', () => {
+  beforeEach(() => {
+    cy.loadPage('/streamed-epub');
+  });
+
+  it('table of content link should update the page content', () => {
+    cy.iframe(IFRAME_SELECTOR)
+      .findByRole('img', {
+        name: "Alice's Adventures in Wonderland, by Lewis Carroll",
+      })
+      .should('exist');
+
+    // Open TOC menu
+    cy.findByRole('button', { name: 'Table of Contents' }).click();
+
+    // Open chapter 1
+    cy.findByRole('menuitem', { name: 'I: Down the Rab­bit-Hole' }).click();
+    cy.wait(100); // wait for the nested iframe to render
+
+    cy.iframe(IFRAME_SELECTOR)
+      .find('.subtitle')
+      .contains('Down the Rab­bit-Hole');
+  });
+
+  it('next and previous button', () => {
```

Comment:
It can be useful to use the "it" to form a sentence for the test names, like `it("should navigate forward and backwards with page buttons")`. Although it's not always possible to phrase it like that, but when we can I think we should try to

Reply:
Yea that makes sense!